usr/share/rear/output/USB/Linux-i386/850_make_USB_bootable.sh

```diff
@@ -59,10 +59,11 @@ case "$usb_filesystem" in
         fi
         ;;
     ("")
-        BugError "Filesystem where the booting related files are on $RAW_USB_DEVICE could not be found"
+        LogPrintError "Could not find a filesystem for $BUILD_DIR/outputfs in /proc/mounts"
+        Error "An ext2/3/4 or vfat filesystem must be mounted for the booting related files on $RAW_USB_DEVICE"
         ;;
     (*)
-        Error "Filesystem $usb_filesystem for the booting related files is not supported"
+        Error "Unsupported filesystem $usb_filesystem for the booting related files on $RAW_USB_DEVICE"
```

Comment:
This part is a bit misleading. The purpose of the code is to determine the filesystem on `$RAW_USB_DEVICE`. But the code does not use `RAW_USB_DEVICE` at all except in messages and comments. It uses `$BUILD_DIR/outputfs` instead. So, it may well happen that `$RAW_USB_DEVICE` is ext3-formatted, but there is a XFS filesystem mounted at `$BUILD_DIR/outputfs` (because of some `OUTPUT_URL` confusion that makes `RAW_USB_DEVICE` out of sync with `OUTPUT_URL`). In that case, the user, when seeing the error message, will check the filesystem on `$RAW_USB_DEVICE` and will see that it is ext3, and will wonder what is ReaR complaining about.

Reply:
I will show both $BUILD_DIR/outputfs and $RAW_USB_DEVICE
cf. https://github.com/rear/rear/pull/3104/files#r1424231529

Reply:
Hopefully sufficiently improved via
https://github.com/rear/rear/commit/5d86ec547e03fc8fa7d635b46f642682662004da